graylog2-web-interface/src/stores/lookup-tables/LookupTablesStore.js

```diff
@@ -19,24 +19,32 @@ const LookupTablesStore = Reflux.createStore({
       count: 0,
       query: null,
     };
+    this.errorStates = {
+      tables: {},
```

Comment:
This should be the same data type as https://github.com/Graylog2/graylog2-server/pull/3795/files#diff-b4db24176b4bf6ff7554108b3773dc89R132, otherwise consumers of the store will get different data types in some scenarios.